lib/node_modules/@stdlib/stats/base/dists/degenerate/mode/src/main.c

```diff
@@ -0,0 +1,36 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/assert/is_nan.h"
+
+/**
+* Returns the mode of a degenerate distribution.
+*
+* @param mu   constant value of the distribution
+* @return     mode, or NaN if input is invalid
```

Comment:
To be consistent with the return value descriptions elsewhere in this package,

```suggestion
* @return     mode
```